js/cex.js

```diff
@@ -1064,15 +1072,14 @@ module.exports = class cex extends Exchange {
             'timeInForce': undefined,
             'postOnly': undefined,
             'side': side,
-            'price': price,
+            'price': this.parseNumber (price),
```

Comment:
Let's use `safeOrder()` here (and feed it with raw trades) to remove `this.parseNumber` calls .